how-to/integrate-with-salesforce/public/js/preload.js

```diff
@@ -1,5 +0,0 @@
-(() => {
```

Comment:
Removing this file would break the integrate with salesforce sample as it needs this static js file.